src/video-player/components/runtime.tsx

```diff
@@ -168,7 +168,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
   };
 
   const getPoster = () => {
-    if (!hasStartedPlayback) {
+    if (!hasStartedPlayback && authoredState.poster !== "") {
```

Comment:
I would change this to `if (!hasStartedPlayback && !!authoredState.poster {` in case `authoredState.poster` is `null`.

The `!!` will return `false` for both `null` and the empty string.

Reply:
Got it -- thanks. Change made.